### PR TITLE
Share a study with custodians

### PR DESCRIPTION
**Pull request by RoriCremer**
kFxa15Hecy

### CODE REVIEW

File: src/sagas/repository.js [hunk 1]
```diff
@@ -274,18 +274,77 @@ export function* getStudyById({ payload }) {
...
+    yield put({
+      type: ActionTypes.EXCEPTION,
+      payload: err,
+    });
+  }
+}
+
+export function* addCustodianToStudy({ payload }) {
+  const studyId = payload.studyId;
+  const custodian = payload.users[0];
```
Comment (ahaessly) [inline src/sagas/repository.js:295]: will the UI ever want to add multiple users in one call?

File: src/actions/app.js [hunk 2]
```diff
@@ -66,6 +50,41 @@ export const { removeReaderFromDataset } = createActions({
...
+  [ActionTypes.GET_STUDY_BY_ID_SUCCESS]: study => study,
+});
+
+export const { getStudyPolicy } = createActions({
+  [ActionTypes.GET_STUDY_POLICY]: policy => policy,
+  [ActionTypes.GET_STUDY_POLICY_SUCCESS]: policy => policy,
+});
+
+export const { addCustodianToStudy } = createActions({
+  [ActionTypes.ADD_CUSTODIAN_TO_STUDY]: (studyId, users) => ({
```
Comment (ahaessly) [inline src/actions/app.js:75]: similar to comment below - but if we are only going to process one user at a time, should we be passing users here?

**Review by ahaessly — APPROVED** [review]
looks good

File: src/actions/app.js (hunk 2 above)
Comment (RoriCremer) [reply src/actions/app.js:75]: I'm going to bring up during standup tomorrow -- wonder if this makes sense as a follow on.
The current UX is just to do one at a time, and I think my desire to keep this parallel to datasets is getting in the way

**Pull request merged**